pulp_rpm/app/constants.py

```diff
@@ -1,26 +1,17 @@
 from types import SimpleNamespace
 
+from pulpcore.plugin.constants import ALL_KNOWN_CONTENT_CHECKSUMS
```

Comment:
This will require pulp_rpm to bump a pulpcore version in dependencies, correct?
If so, coul dwe bump it with this PR to >= 3.9.0.dev, so it is not forgotten?
And we probably need to give a heads up to Katello that RPM 3.9 will require pulpcore 3.9.

Reply:
same as comment above